### DISCUSSION TITLE
Repo template suggestion while using Conductor

### DISCUSSION THREAD

**Discussion by akshatdalton** [Netflix/conductor#3401] [Q&A]
Hi there, I'm new here and I'm happy to share that my team has decided to use the Netflix Conductor but we can't find any useful resources on what directory structure to follow while using Netflix Conductor.

To make it more specific, when I use FastAPI, I try to follow the directory structure like: https://fastapi.tiangolo.com/tutorial/bigger-applications and other resources on GitHub.

Any suggestions would be appreciated!

**Comment by v1r3n** [comment]
Hi @akshatdalton are you using Conductor from source and building the binaries or are you using the published binaries from Conductor and using them?  Happy to work with you on the directory structure.

**Reply by akshatdalton** [reply]
@v1r3n I'm using Conductor from source. Though this is unrelated. I think I couldn't describe the question correctly. I'm talking w.r.t. conductor SDKs that we want to use in our python code.

**Comment by v1r3n** [comment]
Hi @akshatdalton you can use a container we publish if you do not want to build form source (https://github.com/orkes-io/orkes-conductor-community#docker)

As for the python repo structure, you can take a look at https://github.com/conductor-sdk/conductor-python for the examples.
But at a high level, after installing the python client dependencies, you can structure your worker code based on standard python source structure.  Happy to chat more on [slack](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg) if needed.
